form-sharing/2-getSheetsList.js

```diff
@@ -0,0 +1,14 @@
+getValues(
+  '1s7K3kxzm5AlpwiALattyc7D9_aIyqWmo2ubcQIUlqlY',
+  'sheetsList of kobo forms!A:K'
+);
+fn(state => {
+  const koboFormsData = state.references[1];
```

Comment:
If in previous step you stored the information in `state.koboFormsData` 
then you won't need to pull the data from `state.references`